metashare/api/jobs.py

```diff
@@ -181,6 +184,7 @@ def commit_changes_from_org(scratch_org, user, desired_changes, commit_message):
         commit = repository.branch(branch).commit
 
         scratch_org.task.refresh_from_db()
+        scratch_org.task.add_ms_git_sha(commit.sha)
         scratch_org.task.has_unmerged_commits = True
```

Comment:
@wlonk This doesn't seem ideal... but I guess we haven't figured out a better consistent way to know whether a PR can be created from a branch back to the parent branch? I guess without an extra call to check whether they have the same HEAD sha...

Is the plan to just do this manually when we capture changes, and then manually set it back `False` when we find out that the PR is merged?

Reply:
Yes, that's the plan, and yes, it doesn't seem ideal, but absent tracking the whole DAG, it seemed a reasonable shortcut.